### ISSUE TITLE
Registering models from lightning not working (different than pytorch-lightning)

### ISSUE THREAD

**Issue by EladDvash** [allegroai/clearml#1294]
## Describe the bug
When using "import lightning" instead of "import pytorch-lightning" models saved via lightning dont register with clearml
lightning is now the official package for pytorch-lightning since 2.0 

## To reproduce
1. Start from the pytorch lightning example
2. change the imports from `import pytorch_lightning as pl` to `import lightning as pl`
3. add the following code to use lightning checkpoints
```
    from lightning.pytorch.callbacks import ModelCheckpoint
    checkpoint = ModelCheckpoint(
        monitor='valid_loss',
        mode='min',
        save_top_k=1,
        save_weights_only=True,
        auto_insert_metric_name=False,
        filename='best',
        verbose=True
    )
    checkpoint_last = ModelCheckpoint(
        save_last=True,
        save_weights_only=False,
        filename='last',
        enable_version_counter=False,
        every_n_epochs=1,
        save_on_train_epoch_end=True,
        auto_insert_metric_name=False,
        verbose=True
    )
    trainer = pl.Trainer(max_epochs=args.max_epochs, callbacks=[checkpoint, checkpoint_last])

```
4. run the experiment - the models are **not** captured by clearml
5. change the imports back to 
```
import pytorch_lightning as pl
from pytorch_lightning.callbacks import ModelCheckpoint
```
7. run again - the models are now captured by clearml  
## Expected behaviour
Models should be captured when using lightning instead of pytorch-lightning

## Environment
* Server type: self hosted
* ClearML SDK Version: 1.6.4
* ClearML Server Version: 1.13.0
* Python Version: 3.10
* OS: Linux

the issue is in "pytorch_bind.py" starting at line 73 where you check for pytorch-lightning in sys.modules and not also lightning
from there you also only import pytorch-lightning and patch it instead of patching both

**Comment by ainoam** [comment]
@EladDvash your clearml SDK version is somewhat behind - Does this still occur with the latest clearml versions?

**Comment by EladDvash** [comment]
You're right I got confused with installed SDK versions, I wanted to have 1.16 and instead had 1.6 installed
